packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -32,8 +33,10 @@ function VisualEditor() {
 				<WritingFlow>
 					<ObserveTyping>
 						<CopyHandler>
-							<PostTitle />
-							<BlockList />
+							<FocusCapture>
```

Comment:
It took me some time to figure out what's happening here. Is the idea to override the default tab focus behavior such that when focus enters the block list (via tab), focus is shifted forcefully into the selected block? I guess this is so that a block can remain "selected" while the user tabs into the surrounding UI, and can return to the same place when they tab back to the list?

Reply:
Exactly. I'll try to explain it better in the component description.

Reply:
I added more information in both `FocusCapture` and `BlockListBlock` about this behaviour.

Reply:
WritingFlow also handles block navigation (arrow and tabbing in some cases), I feel this component should just be used inside it instead of being a public API.

Reply:
Sure, moving it there seems like a good idea.

Reply:
Moved it to WritingFlow now.